pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -96,6 +97,26 @@ var _ = Describe("Node-labeller config", func() {
 		Expect(cpuModels).To(BeEmpty(), "no CPU models are expected to be supported")
 
 		Expect(cpuFeatures).To(HaveLen(4), "number of features doesn't match")
+		fmt.Println(fmt.Sprintf("cpuFeatures: %v", cpuFeatures))
+	})
+
+	It("should mark all CPU models from domCapabilities as known, regardless of usability", func() {
+		nlController.domCapabilitiesFileName = "virsh_domcapabilities.xml"
+
+		err := nlController.loadDomCapabilities()
+		Expect(err).ToNot(HaveOccurred())
+
+		knownCpuModels := nlController.getKnownCpuModels(nlController.clusterConfig.GetObsoleteCPUModels())
```

Comment:
Do we not have any tests higher up against the original `Run` call?

Reply:
Not that I'm aware of.